README.md

```diff
@@ -1,17 +1,16 @@
 ## System Requirements
 
 ### How to use this app
-1. Obtain a key and code from Trello API, set this in the .env file
-2. Get your board ID from tello and set this in the .env file
-3. Obtain from trello your list id's and set these in the *_list_id variables 
+1. set the mongodb connection string variable
+2. create a database called "DevOps"
+3. create 3 collections as below 
 
 ```bash
-trello_key=
-trello_token=
-trello_boardid=
-todo_list_id=
-doing_list_id=
-done_list_id =
+MONGO_CONN=mongodb+srv://<username>:<password>@corndel.gyf3r.mongodb.net/test?w=majority 
```

Comment:
Worth noting that anyone else who signs up will get a slightly different cluster domain name too, i.e. not `corndel.gyf3r.mongodb.net`